app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,9 +42,6 @@ ns.SearchPage = do (ko
       $('#variablesModal').modal('show')
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
-    
-    setTimeout initModal, 2000 if !window.edscportal
-
```

Comment:
@mightynimble - I believe you were the one who suggested using setTimeout (I could find the comment)  Further, deleting this code completely broke the tour.  The modal does not come up at all now.